packages/suite-desktop-core/src/modules/bridge.ts

```diff
@@ -105,8 +105,9 @@ const shouldUseLegacyBridge = (store: Dependencies['store']) => {
     const newBridgeRollout = store.getBridgeSettings().newBridgeRollout || 0;
     // note that this variable is duplicated with UI
     const NEW_BRIDGE_ROLLOUT_THRESHOLD = 1;
-    const legacyBridgeReasonRollout =
-        !isDevEnv && !skipNewBridgeRollout && newBridgeRollout >= NEW_BRIDGE_ROLLOUT_THRESHOLD;
+    const legacyBridgeReasonRollout = !isDevEnv && newBridgeRollout >= NEW_BRIDGE_ROLLOUT_THRESHOLD;
+
+    if (skipNewBridgeRollout && !legacyRequestedByArg) return false;
```

Comment:
To bypass EAP condition

Reply:
That change looks like quite important, to make the node bridge tests.  :+1:

Reply:
hmm we now have rollout 100%. so we might want to stop using rollout for users in EAP (allowPrerelease) and start using it for normal users. 

would you maybe rework it now in that way that:
- if you are in EAP -> node-bridge always
- if you are not in EAP -> apply rollout
- set rollout to 0 (for now)

Reply:
But that's probably for a separate PR, right?

Reply:
yeah, could be, I just thought it might simplify conditions here a bit. But yes, not needed as part of this.